src/main/resources/default/taglib/t/smartValue.html.pasta

```diff
@@ -22,9 +26,16 @@
 <i:if test="isFilled(payload)">
     <i:local name="signature" value="@computeAuthSignature(apply('%s/%s', type, payload))"/>
     <i:local name="id" value="@id"/>
-    <a id="@id"
-       class="text-black link smart-values-link-js"
-       href="javascript:openSmartValues('@id', '@type','@payload', '@signature')">@label</a>
+    <i:if test="!UserContext.getCurrentUser().getTenantId().equals(tenantId)">
+        <a id="@id"
+           class="text-black link smart-values-link-js"
+           href="javascript:openSmartValues('@id', '@type','@payload', '@signature')">@label</a>
```

Comment:
Hmm, wouldn't it be better to pass the tenant ID to the openSmartValues and then filter there (assuming that there is logic there assuming the actual options), rather than hardcoding here?